### DISCUSSION TITLE
product>libraries\IRremote doe not work with attiny1616

### DISCUSSION THREAD

**Discussion by gparoz** [community/community#121001] [Programming Help]
### Select Topic Area

Bug

### Body

program: example----> TinyReceiver:
works perfectly with NANO on all pins
Does not work with mega tinycore and attiny1616, on any pin!
the library supports attiny1616 in GitHub,
the chosen port, A1 (14) works with
Serial.println(F("START " FILE " from " DATE "\r\nUsing library version " VERSION_TINYIR));
and returns to the serial monitor ----> attiny1616 and pin 14
as well as #define TRACE
which sends a long frame to Serial.print each time a key is pressed
which confirms the functionality of the chip and the pins
but no addresses returned
Serial.print(TinyIRReceiverData.Address, HEX); --------> nothing
nor command
Serial.print(TinyIRReceiverData.Command, HEX);--------> nothing
nothing happens from the library on attiny1616
while the same program works perfectly on NANO and on MH-tiny88

**Comment by LiteBrite82** [comment]
Hi @gparoz ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as [related repositories](https://github.com/topics/attiny1616). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. I will change this over to the Programming Help category as your question may be better answered there. 

Best of luck!

**Comment by gparoz** [comment]
THANKS .

I am not comfortable with computers and I don't understand anything about
this site, can you send me a link to where my request is located?

thanks in advance

I don't understand programming well and would just like to have an example
of reception that works as is usually the case


best regards

**Comment by gparoz** [comment]
additional info: I use 
#define USE_EXTENDED_NEC_PROTOCOL  // Like NEC, but take the 1
6 bit address as one 16 bit value and not as 8 bit normal
and 8 bit inverted value.for reception on ATtiny1616

and I transmit with:
uint16_t sAddress = 0xFF00;
uint8_t sCommand = 0x13;  
from a NANO
